JSQMessagesTests/ControllerTests/JSQMessagesViewControllerTests.m

```diff
@@ -72,7 +69,7 @@ - (void)testJSQMessagesViewControllerSubclassInitProgramatically
     [demoVC endAppearanceTransition];
 
     XCTAssertNotNil(demoVC, @"View controller should not be nil");
-    XCTAssertTrue([demoVC isKindOfClass:[DemoMessagesViewController class]], @"View controller should be kind of class: %@", [DemoMessagesViewController class]);
+    XCTAssertTrue([demoVC isKindOfClass:[JSQMessagesViewController class]], @"View controller should be kind of class: %@", [JSQMessagesViewController class]);
```

Comment:
I changed the test case because `DemoMessagesViewController` does not actually override `messagesViewController`. Failed if I don't change it.